src/main/java/dk/cngroup/lentils/controller/CypherController.java

```diff
@@ -43,9 +40,9 @@ public String newCypher(Model model) {
         return VIEW_CYPHER_DETAIL;
     }
 
-    @GetMapping(value = "/update/{id}")
-    public String updateCypher(@PathVariable Long id, Model model) {
-        Cypher cypher = cypherService.getCypher(id);
+    @GetMapping(value = "/update")
+    public String updateCypher(@RequestParam("cypherId") Long cypherId, Model model) {
+        Cypher cypher = cypherService.getCypher(cypherId);
```

Comment:
Proč jsi to předělal z PathVariable na Param ?

Reply:
upravoval jsem request v html strankach z tohoto tvaru "@{'/progress/skip/'+${cypher.cypherId}}" na tento tvar "@{/progress/stage(cypherId=${cypher.cypherId})}" v navaznosti na Radkovo doporuceni (cypherId=${cypher.cypherId}) je parametr - ale upravim to zpet na PathVariable protoze jde pouzit jen {cyhperId} jako ta Pathvariable - prave to zkousim rozjet :)

Reply:
upraveno